File: app/src/main/java/org/tecrash/crashreport2/db/AppDatabase.java
package org.tecrash.crashreport2.db;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Created by xiaocong on 15/11/14.
 */
@Database(name = AppDatabase.NAME, version = AppDatabase.VERSION, generatedClassSeparator = "_")
public class AppDatabase {
    final static String NAME = "appdb";
    final static int VERSION = 1;
}

File: app/src/main/java/org/tecrash/crashreport2/db/AppDatabase.kt
package org.tecrash.crashreport2.db

import com.raizlabs.android.dbflow.annotation.Database
/**
 * Application database object
 * Created by xiaocong on 15/10/3.
 */

@Database(name = AppDatabase.NAME, version = AppDatabase.VERSION, generatedClassSeparator = "_")
public object AppDatabase {
    const val NAME: String = "appdb"
    const val VERSION: Int = 1
}